tests/Unit/Lang/LangFilesTest.php

```diff
@@ -1,17 +1,17 @@
 <?php
+
 namespace Tests\Unit\Lang;
 
-use Tests\TestCase;
 use Illuminate\Support\Facades\Lang;
 use Illuminate\Support\Facades\App;
 
 class LangFilesTest extends BaseTranslationTest
 {
     public function testAllLangFilesWellFormatted()
     {
-        foreach ($this->locales as $locale) {
-            foreach ($this->getAllLangFilenames() as $langFile) {
-                $this->assertIsArray(Lang::get($langFile));
+        foreach ($this->getAllLangFilenames() as $langFile) {
+            if (!empty($langFile)) {
+                $this->assertInternalType('array', Lang::get($langFile));
```

Comment:
This line is throwing a warning about being deprecated in an upcoming PHPUnit 9 release, which is why I had changed it to `assertIsArray()`

Reply:
Didn't mean to overwrite that.